warehouse/query-core/src/main/java/datawave/query/jexl/nodes/BoundedRange.java

```diff
@@ -54,11 +56,27 @@ public BoundedRange(JexlNode node) {
     
     /**
      * Returns {@value #LABEL}.
-     * 
+     *
      * @return the label
      */
     @Override
     public String getLabel() {
         return LABEL;
     }
+    
+    @Override
+    public boolean equals(Object obj) {
+        if (obj == this) {
+            return true;
+        }
+        if (!(obj instanceof BoundedRange)) {
+            return false;
+        }
+        return JexlStringBuildingVisitor.buildQuery(this).equals(JexlStringBuildingVisitor.buildQuery((BoundedRange) obj));
+    }
+    
+    @Override
+    public int hashCode() {
+        return new HashCodeBuilder().append(JexlStringBuildingVisitor.buildQuery(this)).toHashCode();
```

Comment:
I missed this in the initial review, these calls should be `buildQueryWithoutParse`. Otherwise this merge request is good to go, pending integration testing

Reply:
Updated as requested